packages/xrpl/src/Wallet/index.ts

```diff
@@ -440,7 +440,12 @@ class Wallet {
 
       return memo
     })
-    if (!_.isEqual(decoded, tx)) {
+
+    if (txCopy.TransactionType === 'NFTokenMint' && txCopy.URI) {
```

Comment:
Should this check also live in validateNFTokenMint?

Reply:
Ah yes, I'll add it to the model check too.